### ISSUE TITLE
Failed to decrypt Invalid.Message.Exception (+ fail to synchronize data)

### ISSUE THREAD

**Issue by kunturapu** [AsamK/signal-cli#597]
Hello AsamK.
First of all, I want to thank you for making this possible.

I am having the same issue. I cant synchronize my contacts from .local/share/signal-cli/data/mynumber with the Signal-Desktop.
Im on Ubuntu 20.04 LTS, 64bit.
I do not use a smartphone. So I try to do all manually set-up my contact-list into Signal.
I do not use any other messenger.
Atm I do not see any contacs in my Desktop, because there is an error after editing the data-file.
I start the daemon in the terminal, while Signal-Desktop is open.
Then I go to Settings/Import Contacts in the app...nothing happens: then this happens in the terminal:

`Envelope from: +49xxxx (device: 2)
Timestamp: 161xxxx (2021-04-08T15:16:38.585Z)
Exception: org.whispersystems.libsignal.InvalidMessageException: Message from +49xxxx:2 failed to decrypt; sender ratchet public key 0320411226ab37756xxxxxxx message counter 10
Candidate session 0 failed with 'invalid ciphertext message', had 0 receiver chains (ProtocolInvalidMessageException)
Failed to decrypt message.

Envelope from: +49xxx (device: 2)
Timestamp: 161xxxx (2021-04-08T15:16:38.585Z)
Exception: org.whispersystems.libsignal.InvalidMessageException: Message from +49xxxx:2 failed to decrypt; sender ratchet public key 0320411226ab37756xxxxxxx message counter 11
Candidate session 0 failed with 'invalid ciphertext message', had 0 receiver chains (ProtocolInvalidMessageException)
Failed to decrypt message.

Envelope from: +49xxxx(device: 2)
Timestamp: 161xxxx(2021-04-08T15:16:38.585Z)
Exception: org.whispersystems.libsignal.InvalidMessageException: Message from +49xxxx:2 failed to decrypt; sender ratchet public key 0320411226ab37756xxxxxmessage counter 12
Candidate session 0 failed with 'invalid ciphertext message', had 0 receiver chains (ProtocolInvalidMessageException)
Failed to decrypt message.

Envelope from: +49xx(device: 2)
Timestamp: 161xxxx(2021-04-08T15:16:39.478Z)
Exception: org.whispersystems.libsignal.InvalidMessageException: Message from +49xxxx2 failed to decrypt; sender ratchet public key 0320411226ab37756xxxxxmessage counter 13
Candidate session 0 failed with 'invalid ciphertext message', had 0 receiver chains (ProtocolInvalidMessageException)
Failed to decrypt message.
`
I anonymized my data.
When I edit my contact list, it says there is unexpected errors like "}" , but I a just copying the code as mentioned here:

https://ctrl.alt.coop/en/post/signal-without-a-smartphone/

Thats the way I do it:

   "contactStore" : {
     "contacts" : [ {
       "name" : "name",
       "number" : "+49xxxx"
     }, {
       "name" : "name2",
       "number" : "+49yyyyy",
     }]}

But it doesnt work.
What can I do?

Thank you!
kuntur

**Comment by kunturapu** [comment]
I have seen a closed issue, where sendContacts after a load() helps import contacts into Desktop.
I do not understand the process.
Anyone out there who knows about this?
Thanks.

**Comment by kunturapu** [comment]
Hello, I just tried it again.
I started the daemon, with Signal-Desktop closed.

INFO DaemonCommand - Exported dbus object: /org/asamk/Signal
Envelope from: +49xxx(device: 2)
Timestamp: 1617908972213 (2021-04-08T19:09:32.213Z)
Exception: org.whispersystems.libsignal.InvalidMessageException: Message from +49xxx:2 failed to decrypt; sender ratchet public key 0320411226ab37756xxx message counter 19
Candidate session 0 failed with 'invalid ciphertext message', had 0 receiver chains (ProtocolInvalidMessageException)
Failed to decrypt message.

Can anyone help me?
